lib/features/mailbox/domain/state/subaddressing_mailbox_state.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
+import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox.dart';
+import 'package:tmail_ui_user/features/base/state/ui_action_state.dart';
+import 'package:jmap_dart_client/jmap/core/state.dart' as jmap;
+import 'package:tmail_ui_user/features/mailbox/domain/model/mailbox_subaddressing_action.dart';
+
+class LoadingSubaddressingMailbox extends UIState {}
```

Comment:
Should extend from LoadingState

Reply:
ok ! to be honest i copied it from `lib/features/mailbox/domain/state/subscribe_mailbox_state.dart`, where there is `class LoadingSubscribeMailbox extends UIState {}`
what do you want me to do ?